File: Android/GitViewer/app/src/main/java/nl/sping/gitview/model/GithubRepo.kt
package nl.sping.gitview.model

import com.google.gson.annotations.SerializedName

/**
 * Created by sebastiaan on 2019-05-15
 */

data class GithubRepo(
    @SerializedName("name")
    val name: String,
    @SerializedName("url")
    val url: String
)

File: iOS/GitViewer/GitViewer/Models/GithubRepo.swift
//
//  GithubRepo.swift
//  GitViewer
//
//  Created by Sebastiaan Seegers on 14/05/2019.
//  Copyright © 2019 Sping. All rights reserved.
//

import UIKit

class GithubRepo: NSObject {
  var name: String?
  var htmlURL: String?

  convenience init(RepoName name: String, Url url: String) {
    self.init()
    self.name = name
    self.htmlURL = url
  }
}
